---
title: Software
# bibliography: /software/mybib.bib
---

<br>

Here are some software packages that I have developed/co-developed based on my research.

- **BHMSMAfMRI**: This is an R software package that performs Bayesian hierarchical multi-subject multiscale analysis of function MRI (fMRI) data, or other multiscale data, as described in <span style="color:purple;">Sanyal & Ferreira</span> (<a href="#ref1">2012</a>) using wavelet based prior that borrows strength across subjects and provides posterior smooth estimates of the effect sizes and samples from their posterior distribution. Description and download instructions are available at the package webpage at [https://nilotpalsanyal.github.io/BHMSMAfMRI/](https://nilotpalsanyal.github.io/BHMSMAfMRI/).

- **GWASinlps**: This is an R software package that performs Bayesian non-local prior based iterative variable selection for data from genome-wide association studies (GWAS), or other high-dimensional data, as described in <span style="color:purple;">Sanyal et al.</span> (<a href="#ref2">2019</a>). Description and download instructions are available at the package webpage at [https://nilotpalsanyal.github.io/GWASinlps/](https://nilotpalsanyal.github.io/GWASinlps/).

- **CGEN**: This is an R software package that analyzes case-control data in genetic epidemiology. It provides a set of statistical methods for evaluating gene-environment (or gene-gene) interactions under multiplicative and additive risk models (<span style="color:purple;">Sanyal et al.</span>, <a href="#ref3">2021</a>; <span style="color:purple;">Rochemonteix et al.</span>, <a href="#ref4">2021</a>), with or without assuming gene-environment (or gene-gene) independence in the underlying population. Description and download instructions are available at the package webpage at [https://www.bioconductor.org/packages/release/bioc/html/CGEN.html](https://www.bioconductor.org/packages/release/bioc/html/CGEN.html).
<br>


**SPLC-RAT**: My past colleagues at Stanford University have developed this shiny app based on our joint work on the development and validation of the first risk prediction tool for second primary lung cancer that incorporates comprehensive risk factors including smoking information, medical history, treatment, and tumor characteristics using large population-based data. It is available at [https://splc-risk-prediction.shinyapps.io/SPLC-RiskAssessmentTool/](https://splc-risk-prediction.shinyapps.io/SPLC-RiskAssessmentTool/).

<p>&nbsp;</p>

##### *References:*

<div id="ref1">
Sanyal, Nilotpal, and Ferreira, Marco A.R. (2012). Bayesian hierarchical multi-subject multiscale analysis of functional MRI data. Neuroimage, 63, 3, 1519-1531. doi:10.1016/j.neuroimage.2012.08.041.
</div><br>

<div id="ref2">
Sanyal, N., Lo, M.T., Kauppi, K., Djurovic, S., Andreassen, O.A., Johnson, V.E. and Chen, C.H. (2019). GWASinlps: non-local prior based iterative SNP selection tool for genome-wide association studies. Bioinformatics, 35(1), pp.1-11. doi:10.1093/bioinformatics/bty472
</div><br> 

<div id="ref3">
Sanyal, N., Napolioni, V., de Rochemonteix, M., Belloy, M.E., Caporaso, N.E., Landi, M.T., Greicius, M.D., Chatterjee, N. and Han, S.S. (2021). A Robust Test for Additive Gene-Environment Interaction Under the Trend Effect of Genotype Using an Empirical Bayes-Type Shrinkage Estimator. American journal of epidemiology, 190(9), pp.1948-1960. doi:10.1093/aje/kwab124.
</div><br>

<div id="ref4">
De Rochemonteix, M., Napolioni, V., Sanyal, N., Belloy, M.E., Caporaso, N.E., Landi, M.T., Greicius, M.D., Chatterjee, N. and Han, S.S. (2021). A likelihood ratio test for gene-environment interaction based on the trend effect of genotype under an additive risk model using the gene-environment independence assumption. American journal of epidemiology, 190(1), pp.129-141. American journal of epidemiology, 190(9), pp.1948-1960. doi:10.1093/aje/kwaa132.
</div> 


